src/t8_schemes/t8_default/t8_dpyramid_bits.c

```diff
@@ -1466,6 +1466,8 @@ t8_dpyramid_successor (const t8_dpyramid_t * elem, t8_dpyramid_t * succ,
                        const int level)
 {
   t8_dpyramid_t       parent;
+  /*Suppress Compilerwarnings by setting the level*/
+  parent.level = -1;
```

Comment:
Does it not suffice to set this in t8_dpyramid_child_id_unknown_parent
where we also set the type? It would fit better there.